ucp/_libs/ucx_worker.pyx

```diff
@@ -19,12 +19,56 @@ from ..utils import nvtx_annotate
 logger = logging.getLogger("ucx")
 
 
+cdef _drain_worker_tag_recv(ucp_worker_h handle):
```

Comment:
I think it would be good with a description here.

Reply:
Done in https://github.com/rapidsai/ucx-py/pull/737/commits/3ba02b7a2424ff8aa55f12c82296a147638d2bac .